app/services/rebate_forms_service.rb

```diff
@@ -47,13 +47,24 @@ def create_rebate_form(property)
   end
 
   def create_or_update_property(council)
+    return create_or_update_property_with_valuation_id(council) if @rebate_form_attributes['valuation_id']
+    find_or_create_property_with_no_valuation_id(council)
+  end
+
+  def create_or_update_property_with_valuation_id(council)
     property = Property.find_or_create_by(valuation_id: @rebate_form_attributes['valuation_id'],
                                           rating_year: ENV['YEAR'],
                                           council: council)
     property.update(location: @rebate_form_attributes['location'])
     property
   end
 
+  def find_or_create_property_with_no_valuation_id(council)
+    Property.find_or_create_by(location: @rebate_form_attributes['location'],
+                               rating_year: ENV['YEAR'],
```

Comment:
🐘 Setting the year from an environment variable looks super weird, but I think I heard there's a business reason for this so 🤷‍♂

Reply:
I= need to look into this further but the short answer is yes